packages/client/src/Start.re

```diff
@@ -1,20 +0,0 @@
-[@react.component]
-let make = () => {
-  let ({translationsToString, _}, changeLocale): LocaleContext.t =
-    LocaleContext.use();
-
-  <div className="flex w-screen h-screen justify-center items-center">
-    <div>
-      <LanguageSelect
-        languages=I18n.Locale.allLanguages
-        onChange={language => changeLocale(SetLocale(language))}
-        name="languageSelect"
-      />
-      <Button.Primary
-        className="mt-4 bg-green-400"
-        onClick={_ => ReasonReactRouter.push("/boka-resa")}>
-        {translationsToString(BookTrip_Button)->React.string}
-      </Button.Primary>
-    </div>
-  </div>;
-};
```

Comment:
I wonder if it might be an idea to keep some form of this, perhaps make a skeleton for a Profile-page, to be able to write tests for all the features we add (including the possibility of switching languages). What do you think?

Reply:
Sounds like a good idea! Adding it as an issue in our tracker